Remove one test that fails on musl libc.

    Syntax error: Unknown collating element
    In line 4, byte 3:
    
     `        /[[./.]/]/,`
              ^-- Near here

--- a/tests/custom/00_syntax/21_regex_literals
+++ b/tests/custom/00_syntax/21_regex_literals
@@ -94,31 +94,6 @@
 %}
 -- End --
 
-
-Testing that slashes within character classes are not treated as regex
-literal delimitters.
-
--- Expect stdout --
-[
-	"/[/]/",
-	"/[[./.]/]/",
-	"/[[:alpha:]/]/",
-	"/[[=/=]/]/"
-]
--- End --
-
--- Testcase --
-{%
-	printf("%.J\n", [
-		/[/]/,
-		/[[./.]/]/,
-		/[[:alpha:]/]/,
-		/[[=/=]/]/
-	]);
-%}
--- End --
-
-
 Testing that regex extension macros are substituted only outside of
 bracket set expressions.
 
